pkg/rest/filter/filter.go

```diff
@@ -20,9 +20,8 @@
 package filter
 
 import (
-	"strings"
-
 	restful "github.com/emicklei/go-restful"
+	"net"
```

Comment:
Can you manually update the associated [BUILD.bazel](https://github.com/kubevirt/kubevirt/blob/10239eacdbdae4dd896c6adabab5115e5c62e3cf/pkg/rest/filter/BUILD.bazel) file or run `make generate` adding the changes to this PR?